controllers/main_controller.rb

```diff
@@ -9,3 +9,19 @@
   # combined document back to the client.
   erb :"main/welcome"
 end
+
+MyApp.get "/about" do
+  erb :"main/about"
+end
+
+MyApp.get "/favorites" do
+  erb :"main/favorites"
+end
+
+MyApp.get "/greet/Sumeet" do
```

Comment:
When I deleted those lines, I got four errors all with this message:   1) Failure:
MoreTest#test_links_on_favorites [/Users/melindadeinert/Code/web-toy__simple-application/tests/more_test.rb:40]:
Failed assertion, no message given.